arangod/Pregel/Worker.cpp

```diff
@@ -513,6 +527,9 @@ void Worker<V, E, M>::_finishedProcessing() {
     // count all received messages
     _messageStats.receivedCount = _readCache->containedMessageCount();
 
+    _feature.metrics()->pregelMessagesReceived->count(
+        _readCache->containedMessageCount());
```

Comment:
you can also count them during receiving in line 455 when iterating over messages, not sure what we want

Reply:
The reason I chose this place is because it is where currently `_messageStats` are collected, so the risk of getting inconsistent results is lower.